packages/alpha/package.json

```diff
@@ -6,7 +6,7 @@
   "homepage": "https://github.com/cognitedata/cognite-sdk-js#readme",
   "main": "dist/index.js",
   "types": "dist/src/index.d.js",
-  "version": "0.28.1",
+  "version": "0.28.2",
```

Comment:
we never change package version manually here.
version number is based on the commit title, e.g. fix() bumps a minor version, and so on